### DISCUSSION TITLE
DevRel position

### DISCUSSION THREAD

**Discussion by sergfeldman** [pavlovcik/uad-ui#349] [General]
DevRel is a developer relations manager. 

### As a DevRel you will
- Invite Web3 developers from the Ubiquity database to participate in the Bounty program.
- Consult and answer questions on the Bounty program.

### Requirements to DevRel
- High interest in communication with Web3 developers.
- Ability to communicate effectively in English.
- Business development experience is not mandatory, but will help for this role.

### Terms of the ambassador program
Ambassadors perform their activity according to the instructions provided by the Ubiquity core team.

It is expected that ambassadors will spend about 1-2 hours per day.

The coordination of the ambassador program is performed in the Telegram group https://t.me/+d0ROp0Lc9kU0YTVi

Ambassador roles are not paid, although the high performers will be incentivized by:

1) 20 UBQ (Ubiquity governance tokens)
For each new developer invited by the ambassador that started a bounty.

2) 20 DAI 
For each bounty completed by a new developer invited by the ambassador.

3) Chances to win 1000 DAI
Each completed bounty is a chance to win. 

The incentives are paid once per week.

High-performing ambassadors will be invited to the permanent DevRel role.

### Permanent DevRel role 
- The permanent DevRel role is incentivized by 20% of solved bounties by the new developers invited by the ambassador into the Bounty program.
- The incentive for the invitation of a new developer is valid within 1 year after the new developer started the Bounty program.

**Comment by 0x4007** [comment]
Seems like a decent proposal. Lets start it and iterate.

**Comment by sergfeldman** [comment]
GM
Ubiquity https://ubq.fi/ invites 

1) Experienced Web3 developers to the Bounty program.
https://dao.ubq.fi/ubiquity-bounty-program

2) Ambassadors to the DevRel program (developer relationship managers).
The program is described here (link to Notion page).

**Reply by sergfeldman** [reply]
GM
Ubiquity https://ubq.fi/ invites Developer Relationship managers. 
Part-time, incentives based on the results.
Please, read the description of the role here https://dao.ubq.fi/devrel

**Comment by sergfeldman** [comment]
### DevRel activity 

DevRel is a developer relations manager https://en.wikipedia.org/wiki/Developer_relations 

Invite Web3 developers from the Ubiquity database to participate in the Bounty program.

Consult and answer questions on the Bounty program.

### Requirements to DevRel

High interest in communication with Web3 developers.

Ability to communicate effectively in English.

Business development experience is not mandatory but will help for this role.

### Incentives 

DevRel role is incentivized by 20% of solved bounties by the new developers invited by the ambassador into the Bounty program.
The incentive for the invitation of a new developer is valid within 1 year after the new developer started the Bounty program.

The incentives are paid once per week.

High-performing DevRel can win a share of the 1000 DAI - each completed bounty is a chance to win. 

### General info 

DevRel performs their activity according to the instructions provided by the Ubiquity core team.

It is expected that a DevRel will spend not less than 1-2 hours per day.

The coordination of the DevRel is performed in the Telegram group https://t.me/+d0ROp0Lc9kU0YTVi

**Comment by sergfeldman** [comment]
GM, I found you as a Hackathon participant.

Ubiquity offers dope bounties for developers.
You can solve them yourself or refer your friends.

Both actions are incentivized, the details are here https://dao.ubq.fi/devpool

**Comment by sergfeldman** [comment]
Alvaro:

Ubiquity wants to expand it's devpool 

Know some devs?, invite them and get paid

Do those devs also know devs?, even better!, you can earn form their referrals

**Comment by 0x4007** [comment]
###### Standard

"Are you passionate about open source technology and building developer communities? Join our team as a developer relations manager and help us grow our open source code base!"

"Do you have experience in developer outreach and community building? We're looking for a developer relations manager to help us connect with the open source developer community and bring new talent to our team."

"Do you thrive on connecting with developers and helping them succeed? We're looking for a developer relations manager to join our team and drive our open source initiatives forward."

###### Don Draper

"Are you a master of open source technology and community building? Join our team as a developer relations manager and help us take our code base to new heights."

"Are you a skilled developer relations manager with a knack for connecting with top talent? We're looking for someone like you to help us grow our open source community."

"Do you have what it takes to be a developer relations powerhouse? Join our team and help us build the future of open source technology."
